internal/database/database_listener.go

```
@@ -0,0 +1,406 @@
+	}
+	if l.logger.Enabled(ctx, slog.LevelDebug) {
+		data, err := protojson.Marshal(wrapper)
+		if err != nil {
+			l.logger.ErrorContext(
+				ctx,
+				"Failed to marshal payload",
+				slog.Any("error", err),
+			)
+		}
+		var object any
+		json.Unmarshal(data, &object)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)